src/components/seekbar/_seekbar.scss

```diff
@@ -40,7 +40,13 @@
     border-radius: $progress-bar-border-radius;
     position: absolute;
     transition: height 200ms ease, transform 200ms ease;
-
+    &.chapters {
+      background-color: transparent;
+      .virtual-progress-indicator {
+        height: 8px;
+        transform: translateY(-2px);
+      }
+    }
     .progress {
```

Comment:
this style should move to a new sass file inside progress bar folder and progress bar file should move to its own folder

Reply:
done